src/components/carousel/index.tsx

```diff
@@ -276,12 +280,13 @@ class Carousel extends Component<CarouselProps, CarouselState> {
     }
 
     this.goToPage(nextPageIndex, true);
-
+    
```

Comment:
I suggest to run document formatting to remove all redundant spaces that were added 
If you use VSCode you can run the `Format Document` on the file